src/components/EmptyList.tsx

```diff
@@ -0,0 +1,22 @@
+import { t } from 'i18next'
+import SearchMeta from 'src/types/SearchMeta'
+
+interface IEmptyList {
+  meta?: SearchMeta
+}
+
+export default function EmptyList({ meta }: IEmptyList) {
```

Comment:
Since this component handles the result counts, I'd suggest `SearchResultsInfo` or something that reflects it's broader responsibility.